common/build.gradle

```diff
@@ -19,6 +34,7 @@ tasks.withType(JavaCompile) {
 
 repositories {
     jcenter()
+    mavenCentral()
```

Comment:
Everything resolvable from Maven Central should also be resolvable from JCenter. Did that not prove true for you? Please remove `mavenCentral()` references here and everywhere else if possible.